controller/server/controller.go

```diff
@@ -188,7 +188,12 @@ func (c *Controller) Initialize() {
 	}
 
 	//after InitPersistence
-	c.AppEnv.Broker = env.NewBroker(c.AppEnv)
+	c.AppEnv.Broker = env.NewBroker(c.AppEnv, sync2.NewInstantStrategy(c.AppEnv, sync2.InstantStrategyOptions{
```

Comment:
These are a combo of settings that existed before and some new ones. They currently are not rep'ed in the config file.